### PR TITLE
fix(docs): fix compatibility with IE

### PR DESCRIPTION
**Pull request by layershifter**
This fixes docs compatibility with IE.

### CODE REVIEW

File: docs/app/utils.js [hunk 1]
```diff
@@ -27,6 +35,7 @@ export const semanticUICSSRepoURL = 'https://github.com/Semantic-Org/Semantic-UI
 
 export const scrollToAnchor = () => {
   const anchor = location.hash && document.querySelector(location.hash)
+  const offsetY = window.scrollY || window.pageYOffset
```
Comment (layershifter) [inline docs/app/utils.js:38]: `pageYOffset` is alias for `scrollY`, but it's [supported](https://developer.mozilla.org/en/docs/Web/API/Window/pageYOffset) from IE 9

File: docs/app/utils.js [hunk 2]
```diff
@@ -37,9 +46,9 @@ export const scrollToAnchor = () => {
   if (elementTop === 0) return
 
   // hit max scroll boundaries, stop
-  const isScrolledToTop = scrollY === 0
-  const isScrolledToBottom = scrollY + document.body.clientHeight === document.body.scrollHeight
-  const scrollStep = Math.ceil((Math.abs(elementTop / 8))) * Math.sign(elementTop)
+  const isScrolledToTop = offsetY === 0
+  const isScrolledToBottom = offsetY + document.body.clientHeight === document.body.scrollHeight
+  const scrollStep = Math.ceil((Math.abs(elementTop / 8))) * mathSign(elementTop)
```
Comment (layershifter) [inline docs/app/utils.js:51]: `Math.sign` is [not supported](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Math/sign) in IE at all.

**Comment by levithomason** [comment]
Thanks for the updates.

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.4`.

**Pull request merged**